test/fixtures/merchants.yml

```diff
@@ -11,3 +11,10 @@ merchant_2:
   email: lululumen@yahoo.com
   uid: 5435
   provider: github
+
+
+merchant_3:
+  username: leopoke
+  email: lhifi@yahoo.com
+  uid: 1234
+  provider: github
```

Comment:
GitHub complains there is no newline at the end of this file. Add one? I'm not sure if it breaks anything.